server/modules/module_def.js

```diff
@@ -46,29 +46,31 @@ const read = (path) => {
 
 const evalModule = (contents, name, layoutGeometry, network, game, state) => {
   let classes = {};
+  // A fake require that first checks to see if the require is one of our
+  // per-invocation dependencies. If so, uses that. Otherwise, delegates to
+  // normal require.
+  let fakeRequireInstance = fakeRequire.createEnvironment({
+    network,
+    game,
+    state,
+    wallGeometry: new geometry.Polygon(layoutGeometry.points.map((p) => {
+      return {
+        x: p.x - layoutGeometry.extents.x,
+        y: p.y - layoutGeometry.extents.y
+      };
+    })),
+    debug: debugFactory('wall:module:' + name),
+    globalWallGeometry: wallGeometry.getGeo(),
+    
+    // The main registration function.
+    register: function(server, client) {
+      classes.server = server;
+      classes.client = client;
+    },
+  });
   let sandbox = {
-    // A fake require that first checks to see if the require is one of our
-    // per-invocation dependencies. If so, uses that. Otherwise, delegates to
-    // normal require.
-    require: fakeRequire.createEnvironment({
-      network,
-      game,
-      state,
-      wallGeometry: new geometry.Polygon(layoutGeometry.points.map((p) => {
-        return {
-          x: p.x - layoutGeometry.extents.x,
-          y: p.y - layoutGeometry.extents.y
-        };
-      })),
-      debug: debugFactory('wall:module:' + name),
-      globalWallGeometry: wallGeometry.getGeo(),
-      
-      // The main registration function.
-      register: function(server, client) {
-        classes.server = server;
-        classes.client = client;
-      },
-    })
+    require: fakeRequireInstance,
+    serverRequire: fakeRequireInstance
```

Comment:
> Does this mean a file that mixes client and server code should use let foo = serverRequire(...) for things only needed on the server?

Yes, and this is currently the default, due to the demands of the 'register' function (and single entrypoint).

> But, since we now have multi-file modules, if the server code is in a separate file require or serverRequire can be used.

Yes, I think so.